docs/source/en/big_models.md

```diff
@@ -14,110 +14,205 @@ rendered properly in your Markdown viewer.
 
 -->
 
-# Instantiating a big model
+# Instantiate a big model
 
-When you want to use a very big pretrained model, one challenge is to minimize the use of the RAM. The usual workflow
-from PyTorch is:
+A barrier to accessing very large pretrained models is the amount of memory the require. When you load a pretrained PyTorch model, you usually:
 
-1. Create your model with random weights.
+1. Create a model with random weights.
 2. Load your pretrained weights.
 3. Put those pretrained weights in your random model.
 
-Step 1 and 2 both require a full version of the model in memory, which is not a problem in most cases, but if your model starts weighing several GigaBytes, those two copies can make you get out of RAM. Even worse, if you are using `torch.distributed` to launch a distributed training, each process will load the pretrained model and store these two copies in RAM.
+The first two steps both require a full version of the model in memory and if your model starts weighing several GBs, you may not have enough memory for two copies of the model. This problem is amplified in distributed training environments because each process loads the pretrained model and stores two copies in memory.
 
 <Tip>
 
-Note that the randomly created model is initialized with "empty" tensors, which take the space in memory without filling it (thus the random values are whatever was in this chunk of memory at a given time). The random initialization following the appropriate distribution for the kind of model/parameters instantiated (like a normal distribution for instance) is only performed after step 3 on the non-initialized weights, to be as fast as possible! 
+The randomly created model is initialized with "empty" tensors, which take space in memory without filling it. The random values are whatever was in this chunk of memory at the time. To improve loading speed, the random initialization of the weights from the appropriate distribution (such as a normal distribution for instance) is performed *after* the pretrained weights have been loaded in the model.
```

Comment:
To link with the fast init! It's actually default transformers that call `_init_weights`. Torch's `torch.load` does not use any random initialization!